src/flb_log.c

```diff
@@ -745,6 +750,17 @@ int flb_errno_print(int errnum, const char *file, int line)
     return 0;
 }
 
+int flb_WSAGetLastError_print(int errnum, const char *file, int line)
+{
+#ifdef WIN32
```

Comment:
Would it be better to make the body of the function or the whole function conditional? 
This function is only referenced by `flb_WSAGetLastError` if our target is Windows so I don't think making the whole thing conditional would be a problem.

Reply:
Yep fair assessment. Done in [7a1429c](https://github.com/fluent/fluent-bit/pull/10017/commits/7a1429c720b7ac8aa9d74943b44be7c2743678cf)